usaspending_api/accounts/tests/test_federal_endpoints.py

```diff
@@ -0,0 +1,23 @@
+import pytest
```

Comment:
Am leaving a few comments throughout the PR re: the name `federal` instead of `federal_account`. In the long run, I'm wondering if the shorter names are worth the potential ambiguity/confusion. This is strictly backend stuff, so no need hold up the merge, but what do you think about using `test_federal_account_endpoints.py` as the file name here?